### ISSUE TITLE
centered image not centered in latex

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1366]
This Code

```
#!rst

. image:: images/Eye_of_Horus.*
   :width: 20&#37;
   :align: center
```

is rendered to this:

```
#!latex

{\hfill\includegraphics[width=0.200\linewidth]{Eye_of_Horus.png}\hfill}
```

Which does not result in a centered image in LaTeX, because LaTeX drops a \hfill at the end of a line.

Use \hspace*{\fill} instead of \hfill.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1366
- Originally reported by: Jan Ulrich Hasecke
- Originally created at: 2014-01-29T20:02:22.365

**Comment by tk0miya** [comment]
I reproduce this bug with following reST:

```
.. image:: logo.jpg
   :align: center
* Hello
* World
```

bullet list is needed to reproduce this in my environment.

**Comment by tk0miya** [comment]
MEMO: http://tex.stackexchange.com/questions/45948/what-is-the-difference-between-hspace-fill-and-hfill

**Comment by tk0miya** [comment]
Fixed at eea055d
Thank you for reporting
